src/visual/seriesColor.js

```diff
@@ -41,16 +50,23 @@ export default {
                     data.setItemVisual(
                         idx, 'color', color(seriesModel.getDataParams(idx))
                     );
+                    data.setItemVisual(
```

Comment:
I prefer do not support set callback function for `borderColor` until it is really needed.
This kind of callback function brings some complexity.